examples/basic_mysql_connection/index.js

```diff
@@ -8,13 +8,22 @@ var connection = mysql.createConnection({
 	user: config.user,
 	password: config.password,
 	port: config.port,
-    database: config.database
+  database: config.database
 });
 
 connection.connect();
-
-connection.query('SELECT * FROM todos', function (error, results, fields) {
+let query=['select count(*) as items from todos',
+           'select * from todos where done',
+           'select * from todos where !done',
+					 'select * from todos order by due DESC',
```

Comment:
Here's another interesting case where case-sensitivity comes into play. Notice:

```
mysql> DESCRIBE todos;
+----------+--------------+------+-----+---------+-------+
| Field    | Type         | Null | Key | Default | Extra |
+----------+--------------+------+-----+---------+-------+
| Id       | int(11)      | NO   |     | 0       |       |
| Done     | tinyint(1)   | YES  |     | 0       |       |
| Name     | varchar(255) | NO   |     | NULL    |       |
| StatusId | int(11)      | YES  |     | NULL    |       |
| Due      | datetime     | YES  |     | NULL    |       |
+----------+--------------+------+-----+---------+-------+
5 rows in set (0.00 sec)
```

The column name is `Due`, not `due`. MySQL doesn't care, but someone who's reading your code might (if they're scanning through your code looking for the place where you make a database lookup for that column).